### PR TITLE
Update marketplace profiles

### PR DESCRIPTION
**Pull request by yunjiang29**
* Reduce Marketplace test frequency to `28`
* Remove marketplace from `ec` build
* Added `azure-ipi-marketplace-f28-destructive` from 4.11-4.14

### CODE REVIEW

**Comment by yunjiang29** [comment]
@jianlinliu PTAL

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -1810,8 +1810,8 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-marketplace-f14
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1813]: Do you mind to add one more destructive ci jobs for azure marketplace together in this PR?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1813]: seem like azure get marketplace supported from 4.11

Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1813]: Added

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1813]: Great, thanks.

**Comment by yunjiang29** [comment]
@jianlinliu PTAL again

**Comment by jianlinliu** [comment]
cc @liangxia to review

**Pull request merged**